releasenotes/notes/context-df929fc8dfe148ba.yaml

```diff
@@ -0,0 +1,15 @@
+---
+prelude: >
+    Several deprecations have been made to ``Context`` as we prepare to move
+    active span management out of this class.
+deprecations:
+  - |
+    ``Context.add_span`` is deprecated and will be removed in 0.50.
```

Comment:
Missing deprecation for `clone`, otherwise looks good.

Reply:
Ah good catch!